extensions/ipynb/src/helpers.ts

```diff
@@ -27,6 +27,7 @@ export function getPreferredLanguage(metadata?: nbformat.INotebookMetadata) {
 	return translateKernelLanguageToMonaco(jupyterLanguage || defaultLanguage);
 }
 
+const textDecoder = new TextDecoder();
```

Comment:
Not sure we need to keep creating this every single time, this should make a tiny bit of dfference, specially with larger output counts